### PR TITLE
MAYA-121625: cache to USD: take out 'Parent Scope' from options

### PR DESCRIPTION
**Pull request by seando-adsk**
#### MAYA-121625: cache to USD: take out 'Parent Scope' from options
* Used patter for sub-option to remove the Parent Scope field from the Cache To Usd.

### CODE REVIEW

File: lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py [hunk 1]
```diff
@@ -181,7 +184,7 @@ def fileOptionsTabPage(tabLayout):
     # USD file option controls will be parented under this layout.
     # resultCallback not called on "post", is therefore an empty string.
     fileOptionsScroll = cmds.columnLayout('fileOptionsScroll')
-    mel.eval('mayaUsdTranslatorExport("fileOptionsScroll", "post=all;!animation-data", "' + getCacheExportOptions() + '", "")') 
+    mel.eval('mayaUsdTranslatorExport("fileOptionsScroll", "post={exportOpts}", "{cacheOpts}", "")'.format(exportOpts=kTranslatorExportOptions, cacheOpts=getCacheExportOptions()))
```
Comment (seando-adsk) [inline lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py:187]: Used constant so we have same string for both post and query.

File: lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py [hunk 2]
```diff
@@ -34,6 +34,9 @@
 # Cache options in string format, for MEL mayaUsdTranslatorExport() consumption.
 _cacheExportOptions = None
 
+# The options string that we pass to mayaUsdTranslatorExport.
+kTranslatorExportOptions = 'all;!output-parentscope;!animation-data'
```
Comment (seando-adsk) [inline lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py:38]: Created constant for this string (used in 2 places) and added new string "output-parentscope"

File: lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py [hunk 3]
```diff
@@ -241,7 +244,7 @@ def cacheCommitUi(parent, selectedFile):
 
     # The following call will set _cacheExportOptions.  Initial settings not
     # accessed on "query", is therefore an empty string.
-    mel.eval('mayaUsdTranslatorExport("fileOptionsScroll", "query", "", "mayaUsdCacheMayaReference_setCacheOptions")')
+    mel.eval('mayaUsdTranslatorExport("fileOptionsScroll", "query={exportOpts}", "", "mayaUsdCacheMayaReference_setCacheOptions")'.format(exportOpts=kTranslatorExportOptions))
```
Comment (seando-adsk) [inline lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py:247]: Need the same option string for the query so you don't get back those options (like parent scope).

**Review by ppt-adsk — APPROVED** [review]

File: plugin/adsk/scripts/mayaUSDRegisterStrings.mel [hunk 4]
```diff
@@ -162,9 +162,9 @@ global proc mayaUSDRegisterStrings()
     register("kExportMergeShapesLbl", "Merge Transform and\nShape Nodes:");
     register("kExportNamespacesAnn", "By default, namespaces are exported to the USD file in the following format: nameSpaceExample_pPlatonic1");
     register("kExportNamespacesLbl", "Include Namespaces:");
-    register("kExportParentScopAnn", "Name of the USD scope that is the parent of the exported data.");
-    register("kExportParentScopLbl", "Create USD Parent Scope:");
-    register("kExportParentScopPht", "USD Prim Name");
+    register("kExportParentScopeAnn", "Name of the USD scope that is the parent of the exported data.");
+    register("kExportParentScopeLbl", "Create USD Parent Scope:");
+    register("kExportParentScopePht", "USD Prim Name");
```
Comment (seando-adsk) [inline plugin/adsk/scripts/mayaUSDRegisterStrings.mel:167]: Fixed typo in name

File: plugin/adsk/scripts/mayaUsdTranslatorExport.mel [hunk 5]
```diff
@@ -534,10 +536,12 @@ global proc int mayaUsdTranslatorExport (string $parent,
...
 
-                separator -style "none";
+                if (stringArrayContains("output-parentscope", $sectionNames)) {
+                    separator -style "none";
 
-                textFieldGrp -l `getMayaUsdString("kExportParentScopLbl")` -placeholderText `getMayaUsdString("kExportParentScopPht")`
-                    -annotation `getMayaUsdString("kExportParentScopAnn")` parentScopeField;
+                    textFieldGrp -l `getMayaUsdString("kExportParentScopeLbl")` -placeholderText `getMayaUsdString("kExportParentScopePht")`
+                        -annotation `getMayaUsdString("kExportParentScopeAnn")` parentScopeField;
+                }
```
Comment (seando-adsk) [inline plugin/adsk/scripts/mayaUsdTranslatorExport.mel:544]: Added ability to not add this parent scope field.

**Review by ppt-adsk — APPROVED** [review]

**Pull request merged**